src/Shell/Task/ModelTask.php

```diff
@@ -875,6 +875,16 @@ public function listUnskipped()
     }
 
     /**
+     * Models never have routing prefixes applied.
+     *
+     * @return string
+     */
+    protected function _getPrefix()
```

Comment:
Might be wise to rename this to ```getRoutingPrefix()``` to match the doc block. If table prefixes make it in --routing-prefix and --table-prefix should be the command line options (and the matching protected method names).